showcases/patternhub/pages/foundations/variables/examples.tsx

```diff
@@ -20,7 +20,7 @@ const Example = () => {
 					</li>
 					<li>
 						Don't use it like this in a real app. 💢For example
-						`width: var(--db-spacing-fixed-md)`💥
+						&nbsp;<code>width: var(--db-spacing-fixed-md)</code> 💥
```

Comment:
```suggestion
						<code>width: var(--db-spacing-fixed-md)</code> 💥
```

I can't identify a reason to use a non breaking space here in front of the `code`HTML element.

Reply:
Now I have used a correct space.